### ISSUE TITLE
How do you handle HTTP requests

### ISSUE THREAD

**Issue by icompuiz** [VolleyTKE/BookstoreDAOAndDB#1]
I see a few endpoints in place for responding to get and post on book, but their doesn't seem to be a general purpose framework for responding to HTTP request that specific implementations are built on top of.

You mentioned that you are using JAX-RS.

Here is a tut and example that I found that may be useful.

http://crunchify.com/how-to-build-restful-service-with-java-using-jax-rs-and-jersey/

**Comment by icompuiz** [comment]
Check this out as well

https://jersey.java.net/

**Comment by sigkillchris** [comment]
Yeah, so I am intending on building this using Jersey, then going back and using spring, so I have it built with two different frameworks. I've previously pseudo hacked this together with spring notation, but I'm on the Java servlet jsp kick and I want to go that route before I go to spring. Thank you for your help so far.
